spec/ar2dto/options_spec.rb

```diff
@@ -31,6 +31,12 @@ def self.name
     it "respects the custom class_name" do
       expect(Car.new.to_dto).to be_a(VehicleDTO)
     end
+
```

Comment:
Maybe another test in `spec/ar2dto/record_to_dto/namespaces_spec.rb` may be worth?
What do you think?